test/PerformanceTests/SerializationComparisonsTests/Lib/ODataMessageWriterAsyncPayloadWriter.cs

```diff
@@ -35,6 +35,7 @@ public ODataMessageWriterAsyncPayloadWriter(IEdmModel model, Func<Stream, IOData
         public async Task WritePayloadAsync(IEnumerable<Customer> payload, Stream stream, bool includeRawValues)
         {
             ODataMessageWriterSettings settings = new ODataMessageWriterSettings();
+            settings.EnableMessageStreamDisposal = false;

```

Comment:
why did you add this setting?

Reply:
When this setting is set to true and dispose `ODataMessageWriter`, it will dispose the output stream as well. I disabled it because I wanted to use the output stream after finishing the write. For example, in the unit tests for these benchmarks I rewind the `MemoryStream` after the writer has completed so that I can read the contents and verify that it wrote the right thing.